Makefile

```diff
@@ -26,13 +34,6 @@ OCP_PROVIDER_TEMP_DIR = $(PROVIDER_TEMP_DIR)/insights_local
 # How to execute Django's manage.py
 DJANGO_MANAGE = DJANGO_READ_DOT_ENV_FILE=True $(PYTHON) $(PYDIR)/manage.py
 
-# Docker compose specific file
-ifdef compose_file
-    DOCKER_COMPOSE = $(DOCKER)-compose -f docker-compose.yml -f $(compose_file)
```

Comment:
The way we use the `docker-compose.m1.yaml` is by calling
```
docker compose -f docker-compose.yaml -f docker-compose.m1.yaml
```
This line here is what accomplishes that, so we can't just remove this without some kind of replacement.

Reply:
I put that back in.

What do you think about just adding that file automatically for ARM systems rather than needing to set `compose_file` in the environment?

Reply:
I did it for science: https://github.com/project-koku/koku/pull/3914/commits/6b042d24b761a6db9cf4613410cb8a91b63213ed

Reply:
If that's too magic, I'll just switch back to explicitly requiring the env var and put that back in the docs as well.

Reply:
I'm a-okay with the magic route.

I looked into building multi-arch images before, but the nodes used to build trino and hive use rhel8/podman and I could never figure out if multi-arch was supported on those nodes.